.travis.yml

```diff
@@ -8,7 +8,16 @@ services:
 sudo: required
 
 env:
-- KUBE_VERSION=v1.9.4 CHANGE_MINIKUBE_NONE_USER=true
+  global:
+  - DEPLOY_CONDITION=v1.10 # Don't forget to update here to latest kube minor version
+  - CHANGE_MINIKUBE_NONE_USER=true
+  - MINIKUBE_VERSION=v0.26.0
+  - MINIKUBE_ARGS="--vm-driver=none --bootstrapper=localkube --extra-config=apiserver.Authorization.Mode=RBAC"
+  matrix:
+  - KUBE_VERSION=v1.10.0
+  - KUBE_VERSION=v1.9.4
+  - KUBE_VERSION=v1.8.0
+  - KUBE_VERSION=v1.7.5
```

Comment:
Why don't you use latest minor version?

Reply:
Because minikube has supported the below versions only.

```
$ minikube version
minikube version: v0.26.0
$ minikube get-k8s-versions
The following Kubernetes versions are available when using the localkube bootstrapper:
        - v1.10.0
        - v1.9.4
        - v1.9.0
        - v1.8.0
        - v1.7.5
        - v1.7.4
        - v1.7.3
        - v1.7.2
        - v1.7.0
```